index.html

```diff
@@ -2,6 +2,9 @@
 <head>
     <title>Reviews</title>
     <link rel="stylesheet" href="./client/dist/style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```

Comment:
I see you decided to use an external style sheet!  Interested to talk about this eventually and see 
what the differences between this method and using styled-components.

Reply:
> All looks awesome!! You made a lot of progress. The logic looks like its coming together and nearly done! Going to be cool to see them all put together this week...

Yeah. I'm very excited too. Thanks for the review.